.gitignore

```diff
@@ -1,27 +1,2 @@
-# Logs
-logs
-*.log
-
-# Runtime data
-pids
-*.pid
-*.seed
-
-# Directory for instrumented libs generated by jscoverage/JSCover
-lib-cov
-
-# Coverage directory used by tools like istanbul
-coverage
-
-# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
-.grunt
-
-# node-waf configuration
-.lock-wscript
-
-# Compiled binary addons (http://nodejs.org/api/addons.html)
-build/Release
-
-# Dependency directory
-# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
+.DS_Store
```

Comment:
That's why the commit message reads "gitignore for a mac-friendly node project" :stuck_out_tongue:. From a philosophical point of view, I agree with you. However, from a practical point of view, the reality is that macs are everywhere and this files are annoying.

If you feel strongly about it though, I'll remove it.